examples/capi-quick-start/aws-cluster-calico-crs.yaml

```diff
@@ -1,3 +1,7 @@
+#=================================================================
```

Comment:
These files get published as part of the release where users of it can modify these if needed.

Reply:
Thanks I did not realize that!

How about I remove the warning as part of the release process?

Reply:
I wonder if goreleaser will complain if the files are modified in the repository... Do I need to copy them to an external directory?

Reply:
I think it might... https://goreleaser.com/errors/dirty/